docs/DeveloperGuide.md

```diff
@@ -623,9 +627,74 @@ Priorities: High (must have) - `***`, Medium (nice to have) - `**`, Low (unlikel
 
     Use case ends.
 
+**Policy related Use Cases**
 
+**Use Case 10: Add Policy** 
 
-**Use Case (X): Add Event**
+**MSS**
+
+1. User loads the application
+2. User inputs the command to add a policy
+3. User tags on the details he would like to specify
+4. LTNS switches to the default policy view automatically
+5. LTNS adds the latest policy to the list policy view
+
+    Use Case ends
+
+**Extensions**
+
+* 3a. User attempts to add a Policy which already exists 
+  * 3a1. LTNS displays en error message.
+  
+    Use Case resumes at Step 2
+
+
+**Use Case 11: Assign Policy**
+
+**MSS**
+1. Use loads the application
+2. User inputs the command to assign a existing policy to an existing client
+3. User tags on the details he would like to specify 
+4. LTNS switches to the default client view automatically 
+5. LTNS assigns the specified policy to the specified client
+    
+    Use Case ends
+
+**Extensions**
+* 3a. User inputs an invalid index when specifying the policy 
+    * 3a.1 LTNS displays an error message.
+      
+        Use Case resumes at Step 2
+
+
+
+* 3b. User inputs an invalid index when specifying the client
+    * 3b.1 LTNS displays an error message.
+
+      Use Case resumes at Step 2
+
+
+**Use Case 12: List Policies**
```

Comment:
I think we can avoid confusion and distinguish between view policies and list policies by trying the following:
"Viewing all policies"

Reply:
If adopted, similar implementations might need to be applied for list clients and list events!